libsolidity/analysis/ReferencesResolver.cpp

```diff
@@ -215,22 +215,21 @@ void ReferencesResolver::operator()(yul::FunctionDefinition const& _function)
 
 void ReferencesResolver::operator()(yul::Identifier const& _identifier)
 {
-	bool isSlot = boost::algorithm::ends_with(_identifier.name.str(), ".slot");
-	bool isOffset = boost::algorithm::ends_with(_identifier.name.str(), ".offset");
+	static set<string> suffixes{"slot", "offset", "length"};
```

Comment:
```suggestion
	static set<string> suffixes{"length", "offset", "slot"};
```
Maybe have it sorted?

Reply:
Sorted it not better, I think. According to [this](https://www.cs.usfca.edu/~galles/visualization/RedBlack.html). "offset", "length", "slot" would minimize reordering. But let's ignore this.